ref/Microsoft.Build.Framework/net/Microsoft.Build.Framework.cs

```diff
@@ -1,5 +1,84 @@
 // Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
+namespace Microsoft.Build.Eventing
+{
+    [System.Diagnostics.Tracing.EventSourceAttribute(Name="Microsoft-Build-Profiling")]
+    public sealed partial class MSBuildEventSource : System.Diagnostics.Tracing.EventSource
```

Comment:
One other thing here.  You should just make MSBuildEventSource internal.  You shouldn't need it to be public, and you won't want anyone else building an app that emits msbuild events.